bangazonapi/models/favorite.py

```diff
@@ -6,7 +6,12 @@
 from safedelete.models import SafeDeleteModel
 from safedelete.models import SOFT_DELETE
 
+
 class Favorite(models.Model):
 
     customer = models.ForeignKey(Customer, on_delete=models.DO_NOTHING,)
-    seller = models.ForeignKey(Customer, on_delete=models.DO_NOTHING, related_name='favorited_seller')
+    seller = models.ForeignKey(
+        Customer, on_delete=models.DO_NOTHING, related_name='favorited_seller')
+
+    class Meta:
+        unique_together = ('customer', 'seller')
```

Comment:
You don't have to change this here, but there's a newer/better way to do unique using `UniqueConstraint` .  Could be a good thing to put on your list of things to research for the future.

Reply:
Thanks!  I see that in the docs now.  Commenting here just for future reference. :)

https://docs.djangoproject.com/en/3.1/ref/models/options/#unique-together

https://docs.djangoproject.com/en/3.1/ref/models/constraints/#django.db.models.UniqueConstraint